.pre-commit-config.yaml

```diff
@@ -51,11 +51,16 @@ repos:
     rev: 1.7.0
     hooks:
       - id: bandit
+        name: bandit non-test
         language_version: python3
         exclude: ^test/
+      - id: bandit
+        name: bandit test
+        language_version: python3
+        files: ^test/
         args:
           - -s
-          - "B404,B602,B603,B607"
+          - "B101,B311"
```

Comment:
actually after checking the codes references [here](https://bandit.readthedocs.io/en/latest/plugins/index.html#complete-test-plugin-listing) I believe that we could even clean this list to pass all possible tests /cc @cclauss

Reply:
Please use `--skip` instead of `-s` to make the command more self-documenting for others.

Pytest embraces and encourages the use of `assert` so I always skip `B101` on my own projects.  [`B311`](https://docs.openstack.org/bandit/1.4.0/blacklists/blacklist_calls.html#b311-random) is just a warning not to use the random module for security/cryptographic purposes but if you are not doing that, I think it is OK to skip that one too.